vendor/views/vendor.py

```diff
@@ -305,13 +301,9 @@ def post(self, request, *args, **kwargs):
         processor.authorize_payment()
 
         if processor.transaction_submitted:
-            for order_item_subscription in [order_item for order_item in processor.invoice.order_items.all() if order_item.offer.terms >= TermType.SUBSCRIPTION and order_item.offer.terms < TermType.ONE_TIME_USE]:
-                processor.subscription_payment(order_item_subscription)
-            clear_session_purchase_data(request)
             return redirect('vendor:purchase-summary', pk=invoice.pk)
         else:
-            messages.info(self.request, _(
-                "The payment gateway did not authorize payment."))
+            messages.info(self.request, _("The payment gateway did not authorize payment."))
```

Comment:
We should add a TODO to make the messages configurable for the sites in the settings.

Reply:
Agree, todo added:
```
# TODO: Make message configurable for the site in the settings 
messages.info(self.request, _("The payment gateway did not authorize payment."))
```